openshift/templates/agent/agent-deploy.bc.test.param

```diff
@@ -12,14 +12,14 @@
 # SUFFIX=-bc
 # APP_NAME=orgbook
 # APP_GROUP=aries-vcr
-APPLICATION_DOMAIN=orgbook-agent-test.apps.silver.devops.gov.bc.ca
-APPLICATION_ADMIN_DOMAIN=orgbook-agent-admin-test.apps.silver.devops.gov.bc.ca
+APPLICATION_DOMAIN=agent-test.orgbook.gov.bc.ca
+APPLICATION_ADMIN_DOMAIN=agent-admin-test.orgbook.gov.bc.ca
```

Comment:
Same sync-up comment as for other environments

Reply:
updated